qcodes/instrument/visa.py

```diff
@@ -102,16 +129,19 @@ def set_terminator(self, terminator):
                 eg. '\r\n'.
         """
         self.visa_handle.read_termination = terminator
+        self.visa_handle.write_termination = terminator
```

Comment:
Is this going to break existing drivers?

Reply:
Maybe? 😛 

I'll make sure it only happens for simulated instruments, then we can handle the general case later.